src/v/kafka/tests/topic_recreate_test.cc

```diff
@@ -163,14 +167,6 @@ FIXTURE_TEST(test_topic_recreation, recreate_test_fixture) {
               });
         });
     }).get0();
-
```

Comment:
that's fine, but what is this testing exactly? i don't see any assertions

Reply:
The `cooperative_spin_wait_with_timeout` is an assertion. we wait for certain state to be achieved.